content/en/docs/reference/faq.md

```diff
@@ -228,5 +228,8 @@ spec:
       dapr.io/app-max-concurrency: "value"
 ```
 
+## Q: How to create source repository credential for the function image build process?
 
+A: You may be prompted with "asda" when using `Unsupported type of credentials provided, either SSH private key or username/password is supported (exit code 110)`, which means you are using an incorrect Secret resource as source repository crendital.
```

Comment:
`You may be prompted with "asda"  when using "Unsupported type of" `doesn't sound correct, you mean:
You may be prompted with errors like `Unsupported type of credentials provided, either SSH private key or username/password is supported (exit code 110)`, which means you are using an incorrect Secret resource as the source repository credential.

Reply:
Sorry, I copied the wrong content...🥲